include/ucl++.h

```diff
@@ -100,6 +102,63 @@ class Ucl final {
 		return func;
 	};
 
+	static bool ucl_variable_getter(const unsigned char *data, size_t len,
+			unsigned char **replace, size_t *replace_len, bool *need_free, void* ud)
+	{
+		std::set<std::string> *vars = reinterpret_cast<std::set<std::string> *>(ud);
+		if (vars && data && len != 0) {
+			vars->insert (std::string (data, data + len));
```

Comment:
I'm not familiar with C++11 features yet, but I know just now that everyone prefers it. Thanks!